certified-connectors/HTML To PDF By Pascalcase/settings.json

```diff
@@ -0,0 +1,9 @@
+{

```

Comment:
@VikrantUpadhyay1 Please remove the settings.json file and the Icon.png from this pull request, as they are not required in the open source repository.